content/gamenews

```diff
@@ -1,3 +1,7 @@
+2018-03-07 23-00-00:
```

Comment:
This needs to be updated before this is merged (just pointing out now so we don't forget).